### ISSUE TITLE
NVDA is not seeing some accessible names used in SWT/Eclipse based applications

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2134]
**Reported by mwhapples on 2012-02-28 00:29**
If using the eclipse SWT snippet http://git.eclipse.org/c/platform/eclipse.platform.swt.git/tree/examples/org.eclipse.swt.snippets/src/org/eclipse/swt/snippets/Snippet361.java NVDA is unable to get the name for the AWT canvas inside the SWT canvas although the snippet explicitly sets an accessible name for the control. I have used accprobe to explore the GUI of the snippet and accprobe can find it and give the name. NVDA doesn't seem to give the name either in focus mode or object navigation despite seeing the frame control.

I am using NVDA 2012.1beta1. I know someone else has independently encountered this issue so if I am doing something wrong (either in usage or in installation of NVDA) then its possibly not clear in the documentation as more than me have got this result.

**Comment by leonardder** [comment]
cc @dkager

**Comment by dkager** [comment]
Haha, I saw that one today but didn't investigate.

**Comment by jcsteh** [comment]
The NVDA+f1 dev info for such a control might be enlightening.

**Comment by Adriani90** [comment]
@mwhapples, in case you are still active on Github, could you please provide a NVDA log file by pressing NVDA key+f1 when reproducing this issue? Do you still see this issue happening?

**Comment by Adriani90** [comment]
@mwhapples could you please test now with NVDA 2019.3.1 and the last Eclipse versoin? Is this still an issue?

**Comment by mwhapples** [comment]
Still a problem with NVDA 2020.1 and SWT 4.15.

**Comment by jcsteh** [comment]
Your log file shows the Developer Info (NVDA+f1) for the Windows console. You need to use object navigation to move to the object which should have the name, then press NVDA+f1 at that point and provide the resulting log snippet.

**Comment by Adriani90** [comment]
@mwhapples how is this now with NVDA 2024.3 Beta and SWT 4.32?
Have you made sure that Java Access Bridge is enabled on your system?

**Comment by Adriani90** [comment]
@shadyar, @taymun26 could you also please test whether this issue occurs with NVDA 2024.2 Stable or 2024.3 beta?

**Comment by shadyar** [comment]
It is too old. I suggest to close this issue.

**Comment by Adriani90** [comment]
So did you test? It is still valid even if it‘s old.Von meinem iPhone gesendetAm 10.07.2024 um 16:27 schrieb Shadyar Khodayari ***@***.***>:﻿
It is too old. I suggest to close this issue.

**Comment by seanbudd** [comment]
Closing as invalid/stale. Please open a new issue using the bug report template if this is still valid
